### PR TITLE
Adding steps to manually create Provisioning CR

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25230

@filanov Will you help review this PR

### CODE REVIEW

File: clusters/create_infra_env.adoc [hunk 1]
```diff
@@ -136,6 +137,30 @@ agentserviceconfig.agent-install.openshift.io/agent created
 
 Your CIM service is configured. You can verify that it is healthy by checking the `assisted-service` and `assisted-image-service` deployments and ensuring that their pods are ready and running. Continue with xref:../clusters/create_infra_env.adoc#creating-your-infra-env-with-the-console[Creating your infrastructure environment with the console].
 
+[#manually-create-cr]
+=== Manually create the Provisioning CR
+
+Creating the Provisioning CR enables the hub cluster components that support automated provisioning (the `BareMetalHost` resource). Manually create a `Provisioning` CR to enable services for automated provisioning.
```
Comment (cadawson) [inline clusters/create_infra_env.adoc]: Suggestion: You might be able to consolidate the two intro sentences like:
Run the following command to manually create a `Provisioning` CR to enable services for automated provisioning:

File: clusters/create_infra_env.adoc [hunk 2]
```diff
@@ -136,6 +137,30 @@ agentserviceconfig.agent-install.openshift.io/agent created
 
 Your CIM service is configured. You can verify that it is healthy by checking the `assisted-service` and `assisted-image-service` deployments and ensuring that their pods are ready and running. Continue with xref:../clusters/create_infra_env.adoc#creating-your-infra-env-with-the-console[Creating your infrastructure environment with the console].
 
+[#manually-create-cr]
+=== Manually create the Provisioning CR
```
Comment (cadawson) [inline clusters/create_infra_env.adoc]: Might not be necessary, as it is pretty common, but should we spell out `custom resource` in the first occurrence and/or the title?

**Review by cadawson — APPROVED** [review]
A couple of suggestions, but looks good! Thanks!

File: clusters/create_infra_env.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/create_infra_env.adoc]: @cadawson Thanks for your suggestion! I updated the sentence:

Manually create a Provisioning CR to enable services for automated provisioning by using the following command:

**Comment by dockerymick** [comment]
@cadawson Thanks for your review and approval! I applied your suggestions. Merging this PR

**Pull request merged**